static/app/views/replays/list/replaysErroneousDeadRageCards.tsx

```diff
@@ -154,16 +180,58 @@ function CardTable({
     .map(_ => '1fr')
     .join(' ');
 
+  const emptyLocation = useLocation();
+
+  const emptySearchQuery = {
+    ...emptyLocation.query,
+    cursor: undefined,
+    query: '',
+    sort: '',
+  };
+
   return (
-    <ReplayTable
-      fetchError={fetchError}
-      isFetching={isFetching}
-      replays={replays}
-      sort={undefined}
-      visibleColumns={visibleColumns}
-      saveLocation
-      gridRows={'auto ' + gridRows}
-    />
+    <div>
+      <ReplayTable
+        fetchError={fetchError}
+        isFetching={isFetching}
+        replays={replays}
+        sort={undefined}
+        visibleColumns={visibleColumns}
+        saveLocation
+        gridRows={'auto ' + gridRows}
+      />
+      <Button
+        style={{
+          width: '100%',
+          borderTop: '0',
+          borderTopLeftRadius: '0',
+          borderTopRightRadius: '0',
+          padding: '20px',
+        }}
```

Comment:
we could shove this into `styled()`.

```
const StyledButton = styled(Button)`
  width: 100%;
  border-top: none;

  /* I think this will work, sets 6px on the bottom, 0 on top. If not then see below */
  border-radius: ${p => p.theme.borderRadiusBottom};
  /* border-top-left-radius: 0;
  border-top-right-radius: 0; */

  padding: ${space(3)}:
`;
```

Reply:
uncommented part worked :D